repos/fdbt-site/src/pages/api/deleteOperatorGroup.ts

```diff
@@ -0,0 +1,67 @@
+import { NextApiResponse } from 'next';
+import {
+    // deleteOperatorGroupByNocCodeAndId,
+    getAllProductsByNoc,
+    getOperatorGroupByNocAndId,
+} from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from '../../utils/apiUtils/index';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { getProductsMatchingJson } from 'src/data/s3';
+// import { updateSessionAttribute } from 'src/utils/sessions';
+// import { MANAGE_OPERATOR_GROUP_ERRORS_ATTRIBUTE } from 'src/constants/attributes';
```

Comment:
Should this still be in the code if its commented out?

Reply:
this will be used on a later point, when we complete the delete operator group and hence kept it.